src/binder/assemblybinder.cpp

```diff
@@ -1594,10 +1589,16 @@ namespace BINDER_SPACE
             {
                 BOOL hasHeader = TRUE;
                 IF_FAIL_GO(BinderHasNativeHeader(pNativePEImage, &hasHeader));
+
                 if (!hasHeader)
                 {
-                     pPEImage = pNativePEImage;
```

Comment:
Treating the `pNativePEImage` as a `pPEImage`.  The `pNativePEImage` had a flag set saying it was an trusted native image.  Releasing and rebinding is safer.